vmdb/app/helpers/api_helper/renderer.rb

```diff
@@ -283,19 +283,25 @@ def split_virtual_attribute(attr)
 
     def attr_accessible?(object, attr)
       return false unless object && object.respond_to?(attr)
-      is_reflection = object.class.reflections_with_virtual.keys.collect(&:to_s).include?(attr)
-      is_column = object.class.columns_hash_with_virtual.keys.include?(attr) unless is_reflection
+      is_reflection = object.class.reflection_with_virtual(attr.to_sym)
+      is_column     = object.class.columns_hash_with_virtual.key?(attr) unless is_reflection
```

Comment:
Yeah I started introducing a method like that locally, but was having a hard time with a name.  reflect_on_column seems weird.